packages/sdk/CHANGELOG.md

```diff
@@ -0,0 +1,194 @@
+# [1.34.0](https://github.com/Voltz-Protocol/v2-off-chain-monorepo/compare/v1.33.0...v1.34.0) (2023-05-04)
```

Comment:
why do you add CHANGELOG in commits? This should be autogenerated

Reply:
hmm not sure why this was created here, I removed it and it seems like the new commit didn't generate this